app/src/main/java/mod/hilal/saif/activities/tools/ConfigActivity.java

```diff
@@ -44,8 +44,9 @@ public class ConfigActivity extends Activity {
     public static final String SETTING_SHOW_EVERY_SINGLE_BLOCK = "show-every-single-block";
     public static final String SETTING_USE_NEW_VERSION_CONTROL = "use-new-version-control";
     public static final String SETTING_USE_ASD_HIGHLIGHTER = "use-asd-highlighter";
+    public static final String SETTING_EXECUTE_SHELL_SCRIPT = "execute-shell-script";
     public static final String SETTING_SKIP_MAJOR_CHANGES_REMINDER = "skip-major-changes-reminder";
-    public static final String SETTING_BLOCKMANAGER_DIRECTORY_PALETTE_FILE_PATH = "palletteDir";
+    public static final String SETTING_BLOCKMANAGER_DIRECTORY_PALETTE_FILE_PATH = "paletteDir";
```

Comment:
Why? This will only break compatibility with older Sketchware Pro versions.

Reply:
the pallete part ill change it back lol i just thought it was a spelling error sorry